homework1/tests/test_task3.py

```diff
@@ -0,0 +1,19 @@
+from typing import Tuple
+
+import pytest
+
+from homework1.task03 import find_maximum_and_minimum
+
+
+@pytest.mark.parametrize(
+    ["file_name", "expected_result"],
+    [
+        ("tests/test_data/task3_test1.txt", (-1000, 1000)),
+        ("tests/test_data/task3_test2.txt", (-15, 14)),
+        ("tests/test_data/task3_test3.txt", (0, 0)),
```

Comment:
What could be done to not test the function with predefined files (they could be large and not suitable to push to the repository or you might came up with huge number of test cases)?

Reply:
🤔

Reply:
I added random numbers generator for the file

Reply:
Done

Reply:
It is not exactly what I meant. You can split the task into two functions - one reads a file and the other finds min and max. This way you can test the main functionality without creating any files.